runs/mqttsub.py

```diff
@@ -1419,11 +1419,41 @@ def on_message(client, userdata, msg):
                     f = open('/var/www/html/openWB/ramdisk/evuv3', 'w')
                     f.write(msg.payload.decode("utf-8"))
                     f.close()
-            if (msg.topic == "openWB/set/evu/HzFrequenz"):
+            if ((msg.topic == "openWB/set/evu/HzFrequenz") or (msg.topic == "openWB/set/evu/Hz")):
                 if (float(msg.payload) >= 0 and float(msg.payload) <= 80):
                     f = open('/var/www/html/openWB/ramdisk/evuhz', 'w')
                     f.write(msg.payload.decode("utf-8"))
                     f.close()
+            if (msg.topic == "openWB/set/evu/WPhase1"):
+                if (float(msg.payload) >= -1000 and float(msg.payload) <= 1000):
+                    f = open('/var/www/html/openWB/ramdisk/bezugw1', 'w')
+                    f.write(msg.payload.decode("utf-8"))
+                    f.close()
+            if (msg.topic == "openWB/set/evu/WPhase2"):
+                if (float(msg.payload) >= -1000 and float(msg.payload) <= 1000):
+                    f = open('/var/www/html/openWB/ramdisk/bezugw2', 'w')
+                    f.write(msg.payload.decode("utf-8"))
+                    f.close()
+            if (msg.topic == "openWB/set/evu/WPhase3"):
+                if (float(msg.payload) >= -1000 and float(msg.payload) <= 1000):
+                    f = open('/var/www/html/openWB/ramdisk/bezugw3', 'w')
+                    f.write(msg.payload.decode("utf-8"))
+                    f.close()
```

Comment:
Leider ist diese Datei größtenteils kein gutes Vorbild... Kürzer wäre:
```python
wphase_match = re.match("^openWB/set/evu/WPhase([123])$")
if wphase_match is not None:
    files.evu.powers_import[int(wphase_match.group(1)) - 1].write(float(msg.payload.decode("utf-8")))
```
Warum filterst du Werte raus die kleiner als -1000 oder größer als 1000 sind? Oder ist die Angabe nicht in Watt?

Selbst wenn du was filterst - leider ist auch da die Datei kein gutes Vorbild. Ein
```python
log.warning("Ignoring value %s for topic %s (out of range)", msg.payload, msg.topic)
```
Könnte Leuten helfen, wenn es mal Probleme gibt.


Der ganze Kommentar gilt natürlich auch für den power factor. Unten nach gleichem Muster.

Reply:
Danke für das feedback - wieder was gelernt.
Änderungen sind gemacht für die neuen topics.
Filter für die neuen topics sind entfernt.
Bestehenden code habe ich belassen.